src/components/snapshots/routes/helpers.ts

```diff
@@ -9,7 +9,12 @@ export const getSnaphotList = async () => {
 
   for (const dirent of dirents) {
     if (!dirent.isDirectory()) continue
-    if (!fsSync.existsSync(path.join(SNAPSHOT_FOLDER, dirent.name, `${SNAPSHOT_FILE_NAME}.json`)))
+    if (
+      !(
+        fsSync.existsSync(path.join(SNAPSHOT_FOLDER, dirent.name, `${SNAPSHOT_FILE_NAME}.json`)) ||
+        fsSync.existsSync(path.join(SNAPSHOT_FOLDER, dirent.name, `database.dump`))
+      )
+    )
```

Comment:
I suggest a comment at the top, I'm not 100% sure of what this is doing. Is it trying to find a snapshot for the given name, and **if snapshot not found** will check if some `datase.dumb` folder exists in the same snapshots folder?

Reply:
This is just changing what Andrei already had. It would check that `snapshot.json` was present and only consider it an actual snapshot to list if so. This just extends that to accept `snapshot.json` *or* `database.dump`.

Reply:
Oh I see it's only adding to the list of snapshots to be used